src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -1090,6 +1090,32 @@ impl_install_tree (RpmOstreeTreeComposeContext *self,
   return TRUE;
 }
 
+static gboolean
+create_rpmdb_variant (int              rootfs_dfd,
+                      GVariant       **out_variant,
+                      GCancellable    *cancellable,
+                      GError         **error)
+{
+  g_autoptr(GPtrArray) pkglist = NULL;
+  g_autoptr(RpmOstreeRefSack) refsack = NULL;
+  if (!rpmostree_get_pkglist_for_root (rootfs_dfd, ".", &refsack,
+                                       &pkglist, cancellable, error))
+    return FALSE;
+
+  GVariantBuilder pkglist_v_builder;
+  g_variant_builder_init (&pkglist_v_builder, (GVariantType*)"av");
```

Comment:
I think this should be `a(stss)` or so - the individual variants shouldn't vary.